### DISCUSSION TITLE
New here.  I'm confused!

### DISCUSSION THREAD

**Discussion by topsquarks** [MiKTeX/miktex#1185] [General]
Okay, maybe I'm getting into something I didn't expect.  I downloaded and installed MikTeX assuming I'd get an offline LaTeX editor.  But how do I use it?  All I can find out how to do is run the console.

Thanks!
-Dan

**Comment by vitorlppf** [comment]
From the console you should be able to see a button for TeXworks under the settings option, above the terminal button. You can also configure TeXworks as the default program to edit .tex files with.

-Vitor

**Comment by topsquarks** [comment]
I'm sorry, I should have posted.  I may come back to MikTeX but I found out
my current Tex based word processor already has TikZ capacity.  I'm looking
at that to see how well it works for me.  Thank you for the information,
though. I appreciate the response.

-Dan
